pyflakes/messages.py

```diff
@@ -116,6 +116,22 @@ def __init__(self, filename, loc, name):
         self.message_args = (name,)
 
 
+class DuplicateDictionaryKey(Message):
+    message = 'dictionary key %s duplicated'
```

Comment:
That makes perfect sense. Thanks for being conscious of that @jayvdb. @geokala can you fix this up?